local-config/fabric/README.md

```diff
@@ -132,8 +49,17 @@ Generate the test-network connection-full.json file, which describes the fabric
 cd connect/local-config/fabric/conf
 ./ccp-generate.sh
 ```
+Note:  If you have redeployed test-network, delete all the .id files under connect/local-config/fabric/conf/wallet, as these will no longer work with your new test-network deployment.  They will be regenerated when the connect fabric client starts.
+
+## Configure LinuxForHealth connect
+Configure LinuxForHealth connect to use the test-network Docker network.  This is required for contract network connectivity with the connect NATS server.  Change the network name in connect/docker-compose.yml to `fabric_test`:
+```shell
+networks:
+  main:
+    name: fabric_test
```

Comment:
Is the `fabric_test` network defined in the docker-compose.yml?

Reply:
no, it's defined in the HF test-network docker-compose files.